.github/ISSUE_TEMPLATE/crash-report.md

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
For this file in general, although the name is the same as the sodium one, the name should use a _ rather than a - in the name to be consistent with the rest of the files.